lib/fedora_migrate/repository_migrator.rb

```diff
@@ -52,6 +37,32 @@ def get_source_objects
 
     private
 
+    def migrate_object source
+      Logger.info "Migrating source object #{source.pid}"
+      result = begin
+        FedoraMigrate::ObjectMover.new(source, nil, options).migrate
+      rescue NameError => e
+        "The most likely explanation is that your target is not defined"
+        error_message(e)
+      rescue StandardError => e
+        error_message(e)
```

Comment:
I revamped both the migrate_object and migrate_relationship methods to be more uniform.